pandas/core/frame.py

```diff
@@ -6177,14 +6177,14 @@ def stack(self, level=-1, dropna=True):
 
     def explode(self, column: Union[str, Tuple]) -> "DataFrame":
         """
-        Transform each element of a list-like to a row, replicating the
-        index values.
+        Transform each element of a list-like to a row, replicating index values.
 
         .. versionadded:: 0.25.0
 
         Parameters
         ----------
         column : str or tuple
+            Column(s) to explode.
```

Comment:
```suggestion
            Column to explode.
```

The tuple refers to a MultiIndex name, not to the ability to pass multiple column names